climakitae/explore/uncertainty.py

```diff
@@ -696,14 +696,14 @@ def get_warm_level(warm_level, ds, multi_ens=False, ipcc=True):
 
     Parameters
     ----------
-    warm_level: float or int
+    warm_level : float or int
```

Comment:
@Tianchi-Liu I am wondering here if we need to remove 4.0 as an option in this list, and add 2.5.